en/docs/quick-starts/query-saml-assertions-sample.md

```diff
@@ -0,0 +1,169 @@
+# Query SAML2 Assertions
+
+This page guides you through querying dynamic or existing [SAML2 assertions](insertlink) for a sample application using standard request messages via WSO2 Identity Server. 
+
+----
+If you have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../guides/login/query-saml-assertions" target="_blank" rel="nofollow noopener">I have my own application</a>
+
+----
+
+## Set up the sample application
+
+{!fragments/travelocity.md!}
+
+----
+
+## Enable querying saml assertions
+
+1. Click **Service Providers > List** and click **Edit** on the Travelocity service provider you created. Edit the SAML2 configuration. 
+
+2. Select **Enable Assertion Query Request Profile** to enable saml assertions. 
+
+3. Click **Update**.
+
+-----
+
+## Persist assertions to the database
+
+Optionally, you can use a custom assertion builder that enables persisting assertions in the database for this profile. 
+
+1.  Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.
+
+2.  Add the following configuration.
+
+    ``` toml
+    [saml.extensions] 
+    assertion_builder= "org.wso2.carbon.identity.sso.saml.builders.assertion.ExtendedDefaultAssertionBuilder"
+    ```
+
+3. Restart the server. 
+
+-----
+
+## Query SAML assertions
+
+1. Access the Travelocity application via `http://wso2is.local:8080/travelocity.com/index.jsp`
+
+2.  Start the SAML Tracer.
+
+    !!! tip
+        If you do not have a SAML tracer already, install a SAML Tracer (plugin/application/browser extension) that enables searching assertions.
+
+3.  Log in using administrator credentials (admin:admin).
+
+    When a user logs in, the created assertion will be persisted in the SAML Tracer.
+
+4. Copy the attribute **ID** value that is located inside the tag `<saml2:Assertion>` of the created assertion. You can use this ID to query the assertion using a `AssertionIDRequest`.
```

Comment:
Don't we need to have the same information after this line to exist in query-saml-assertions.md page also. In both the classes we are using the same java client, right? Shall we move this to a fragment and re-use it in both places?